### PR TITLE
add coverage

### PR DESCRIPTION
**Pull request by malaffoon**
Add validation.
Add ETS samples.

### CODE REVIEW

File: model/src/main/java/rdw/model/XmlUtils.java [hunk 1]
```diff
@@ -1,44 +1,78 @@
...
-            marshaller.marshal(report, os);
-        } catch (final JAXBException e) {
-            throw new RuntimeException(e);
+
+    private static JAXBContext context;
+    private static Schema schema;
+
+    private static JAXBContext getContext() throws JAXBException {
+        // although not thread-safe, the worst that happens is double instantiation
+        if (context == null) {
```
Comment (malaffoon) [inline model/src/main/java/rdw/model/XmlUtils.java]: this significantly improves performance when marshaling lots of samples

File: model/src/main/resources/TestResultsTransmissionFormat.xsd [hunk 2]
```diff
@@ -23,9 +23,9 @@
             <!-- not expected to be used for open source, but may be needed for internal purposes -->
             <xs:attribute name="handScoreProject" type="xs:unsignedInt"/>
             <!-- new fields requested for open source -->
-            <xs:attribute name="assessmentType" use="required"/>
```
Comment (malaffoon) [inline model/src/main/resources/TestResultsTransmissionFormat.xsd]: These changes are from grabbing the latest copy of xsd from SB website

File: model/src/test/resources/TDSReport.sample.xml [hunk 3]
```diff
@@ -46,7 +46,7 @@
   </Examinee>
   <Opportunity server="562299-SBASQL8" database="CASBAC_SHARD_2013sp" clientName="California"
                key="71A3EE01-F215-4CCD-B74D-DF1097A01A0C" oppId="1855629"
-               startDate="2014-04-14T10:49:08.437" status="completed" validity="valid" completeness="complete"
```
Comment (malaffoon) [inline model/src/test/resources/TDSReport.sample.xml]: I had to make these changes (by hand) for the SB sample to pass validation.

Comment (rcordeau) [inline model/src/test/resources/TDSReport.sample.xml]: Are you going to have to change every report by hand as they come in?

Comment (malaffoon) [inline model/src/test/resources/TDSReport.sample.xml]: Yeah.

What's the concern? Scalability?

Comment (rcordeau) [inline model/src/test/resources/TDSReport.sample.xml]: Yeah. Seems like something a computer could do....

Comment (malaffoon) [inline model/src/test/resources/TDSReport.sample.xml]: Hmm, that's a good idea. I'll look into that.

**Pull request merged**